frontend/src/components/BoardUnit.tsx

```diff
@@ -20,16 +20,14 @@ const BoardUnit: React.FunctionComponent<Props> = ({ unit }: Props) => {
           onMouseEnter={() => setHover(true)}
```

Comment:
Do we still need this if the trigger has been set to `hover`?

Reply:
Actually, yes! It does not really matter what the trigger is, I believe it's covered by some other element.